unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -3009,8 +3101,12 @@ contending and disputing about them, but can never know,
 while they are from the thing within: for that mind is not the heir,
 nor must it inherit the kingdom of God.
 
-==== POSTSCRIPT
+====
+
+[.old-style]
+=== Postscript
```

Comment:
Okay, so this was a really weird part with the postscript stuff. I couldn't put it [.postscript] wrap because I couldn't really tell where it ended, which I decided finally that it ended at the end of the chapter, but a [.numbered-group] ends there as well and nothing was working with using the ==== termination from both things at the end, if that makes sense. So unless the postscript ends earlier, I couldn't figure out any other way to make this look okay. But it all seemed to me to be part of the postscript, and I couldn't tell even from the PDF.

Reply:
I think you did basically the right thing.  There are a few documents where `Postscript` is really just a section heading, not a PS for a letter.

The only thing I'd say here is I would remove the `[.old-style]`.  For two reasons: 1) an `[.old-style]` designation should really only be used on a heading that is _multi-line_ using the `/` character to separate the lines.  It just so happens that `.old-style` also _centers_ the text.  But if you just want something centered, it's better to use the `.centered` class.

All that said, I would probably lean towards not worrying about how things are centered in the ODT, so here, if I were doing it, i would not make it centered, just a plain `===` seems right to me.